draft-ietf-quic-tls.md

```diff
@@ -232,7 +232,7 @@ A simplified TLS 1.3 handshake with 0-RTT application data is shown in
 
     ClientHello
    (0-RTT Application Data)
-   (end_of_early_data)        -------->
+   (EndOfEarlyData)          -------->
```

Comment:
This goes with Finished in the second flight now which, handily, matches the QUIC version of the diagram below.